### DISCUSSION TITLE
Cotton Bale collect mode

### DISCUSSION THREAD

**Discussion by fudoido** [Courseplay/Courseplay_FS22#1574] [Ideas]
Just wondering if this mode is operational also for cotton bales with the module X semi loader trailer, or if there are plans to make it work in a possible future?
I am aware this will give lot more work than the other Bale loader trailers, since this one can only pickup bales by reversing.....
Just wondering if it already works or if its on a to-do list for the (not so soon) future?

**Comment by Tensuko** [comment]
I don't know which tool that is.
If that is the reverse one, that won't happen.

**Reply by fudoido** [reply]
yes its the truck trailer that needs to load by reversing, taking only 2 bales each time.
its a shame it wont be possible to make it work for cotton bales, as far i have read its one of the most profitable crops, im sure lots of people would try to do cotton if they had courseplay help, specially in the new spring creek 32x map :P

**Reply by Tensuko** [reply]
If it is just about money, any map silage bales will do the trick...

x32 maps doesn't matter for us.
Its unrealistic in a game, even x16 is insane already.

**Reply by fudoido** [reply]
i know silage is a good money maker also. was mostly curious about the option of making it work with cotton bales. and only mentioned the map size to make the importance of the help of courseplay and autodrive, not really a map i can spend many time playing also, its too big for single player (even a 16x is, as you mentioned).
that is about it, i was just curious if cotton bales could be somehow implemented in courseplay, but as i said in the beggining, it would be hard since they must be loaded by reversing the trailer, which will require alot more calculations and programming inside courseplay, meaning its not a priority for the team to deal with.
